src/uu/more/src/more.rs

```diff
@@ -293,14 +293,11 @@ impl<'a> Pager<'a> {
     fn should_close(&mut self) -> bool {
         self.upper_mark
             .saturating_add(self.content_rows.into())
-            .eq(&self.line_count)
+            .ge(&self.line_count)
```

Comment:
Is there a reason why this uses `ge` instead of the `>=` operator?

Reply:
Nothing...just happened to use it I guess.